test/quantization/test_workflow_module.py

```diff
@@ -1384,7 +1419,8 @@ def _test_backward_per_channel_cachemask_impl(self, device):
             X.requires_grad_()
             Y_prime = torch.fake_quantize_per_channel_affine(
                 X, scale, zero_point, axis, quant_min, quant_max)
-            dout = torch.rand(X.shape, dtype=torch.float).to(device)
+            # dout = torch.rand(X.shape, dtype=torch.float).to(device)
+            dout = torch.rand(X.shape, dtype=float_type).to(device)
```

Comment:
should this be `torch.rand_like(X, dtype=float_type, device=device)`?

Reply:
I can make this change although I'm wondering if there is a functional difference? Is the rand_like considered a best practice?

Reply:
i think mostly matter of conciseness / style + maybe some savings on creating a random tensor on the target device directly (actually not sure if it's supported on all devices?)